tests/test_parsing.py

```diff
@@ -164,7 +237,11 @@ def test_defendant_appeared(self, index, hearing_index, expected):
             (1, "Roe, Jean", "05/06/2020"),
             (2, "Noe, Ann", "01/22/2020"),
             (3, "Lewis, Lois", None),
+            (4, "Tenant, David", "12/08/2018"),
+            (5, "SEE, LES", "11/16/2019"),
             (6, "Ant, Ten", "01/02/2020"),
+            # (7, "Ant, Ten", "01/08/2020"),
```

Comment:
Not sure what the expected dates should be here. In case J2-CV-20-000021 (example_7.html), there are two case events that have "Served" dates: the "Service" case event and the "Writ" case event. The logic in hearing.py's `was_defendant_served` method uses the most-recent "Served" date found, which in this case is for the "Writ" event. Is that correct? Or should `was_defendant_served` always return the "Service" event's date?

Reply:
I think it should target "Service" case events specifically. They both have RCDSE# headers, but I think that ideally "Service" case events don't overlap with Writs.

Reply:
Thanks, @jtcasper. I created #30 for the issue.